tensorflow/docs_src/community/documentation.md

```diff
@@ -276,7 +276,7 @@ __init__.py:
     # Otherwise import symbols directly
     from tensorflow.some_module.some_other_file import some_symbol
 
-    from tensorflow.platform.all_util import remove_undocumented
+    from tensorflow.platform.util.all_util import remove_undocumented
```

Comment:
This still gives a:

`ImportError: No module named platform.util.all_util`

looks like that "platform" should be replaced with "python"

`from tensorflow.python.util.all_util import remove_undocumented`